frontend/packages/client/src/App.sass

```diff
@@ -278,8 +279,9 @@ hr
 	border-radius: 133.333px
 	max-height: 40px
 .blockies
-	border-radius: 133.333px
-	border: 4px solid #F2F2F2
+	border-radius: 50%
+	&.blockies-border
+		border: 4px solid #F2F2F2
```

Comment:
member list is using 'blockies' class and does not have a thick border:

<img width="453" alt="Screen Shot 2022-06-14 at 12 33 13" src="https://user-images.githubusercontent.com/7526740/173617691-f5c08abb-91f8-4e94-adb2-66de064c40d2.png">

created `blockies-border` to be used with `blockies` for header and card components